src/server/controllers/userController.ts

```diff
@@ -67,6 +80,12 @@ export const registerUser = async (
       message: `User ${user.userName} was registered sucessfully.`,
     });
   } catch (error) {
-    next(error);
+    const customError = new CustomError(
+      400,
+      error.message,
+      "Error with register user."
```

Comment:
Revisa todo lo que son mensages que se devuelven al front para no dar información del tipo de error. No dar feedback de que la contraseña no es válida, dar solo feedback de que los datos son incorrectos.

"User or password not valid"

Reply:
si exacto